src/Knp/Menu/Renderer/Renderer.php

```diff
@@ -2,33 +2,33 @@
 
 namespace Knp\Menu\Renderer;
 
-if (!defined('ENT_SUBSTITUTE')) {
-    define('ENT_SUBSTITUTE', 8);
+if (!\defined('ENT_SUBSTITUTE')) {
```

Comment:
This constant was introduced with php 5.4. I think, we can safely remove that block.